biahub/segment.py

```diff
@@ -200,17 +233,17 @@ def segment_cli(
     )
 
     # Estimate resources
-    num_cpus, gb_ram_request = estimate_resources(shape=segmentation_shape, ram_multiplier=20)
+    num_cpus, gb_ram_request = estimate_resources(shape=segmentation_shape, ram_multiplier=10)
```

Comment:
This seems like a lot of funny math to estimate the resources. Does `estimate_resources` not do everything you need? For example, it now allows for specifying the max number of CPUs.

Reply:
I'd suggest leaving reasonable defaults in the code and using an sbatch file to tune these parameters as needed for specific reconstructions or depending on the cluster usage